.github/workflows/codemeta.yaml

```diff
@@ -0,0 +1,46 @@
+name: codemeta
+
+on:
+  push:
+    paths:
+      - codemeta.json
+      - .github/workflows/codemeta.yaml
+    tags:
+      - 'v*'
```

Comment:
This changes the semantics (AND) compared to the "old" validate workflow. Please comment on this in the commit text body. :-)

Reply:
thx, ye, will revisit this. thought it was OR logic.